src/Access/ContextAccess.cpp

```diff
@@ -756,11 +756,11 @@ bool ContextAccess::checkAdminOptionImplHelper(const Container & role_ids, const
                 show_error(ErrorCodes::ACCESS_DENIED,
                            "Not enough privileges. "
                            "Role {} is granted, but without ADMIN option. "
-                           "To execute this query it's necessary to have the role {} granted with ADMIN option.",
+                           "to execute this query, it's necessary to have the role {} granted with ADMIN option.",
```

Comment:
Here the word `to` is the first word of the new sentence, so it should be `To` in fact?

Reply:
Thank you, I've missed this!